pkg/util/metadata.go

```diff
@@ -46,7 +47,7 @@ func WriteMetaDataFile(vol *csiapi.MetaData) error {
 	}
 
 	metaPath := filepath.Join(vol.Path, csiapi.MetaDataFileName)
-	return WriteFile(metaPath, b, 0600)
+	return WriteFile(metaPath, b, (0600 | os.ModeSticky))
```

Comment:
This file is owner-readwrite-only. Is that correct?

And repeat comment about use of stickybit with files.